app/lib/bk/compat/parsers/bitrise/steps.rb

```diff
@@ -15,11 +15,42 @@ def translator(type, inputs)
           send("translate_#{type.downcase.gsub('-', '_')}", inputs)
         end
 
+        def translate_brew_install(inputs)
+          return '# Invalid brew-install configuration!' unless inputs.include?('verbose_log')
+
+          generate_brew_install_command(inputs)
+        end
+
+        def generate_brew_install_command(inputs)
+          if inputs.fetch('packages', false)
+            install_type = inputs.fetch('upgrade', false) ? 'reinstall' : 'install'
+            "brew #{install_type}#{inputs['verbose_log'] ? ' -vd' : nil} #{inputs['packages']}"
+          elsif inputs.fetch('use_brewfile', false) && inputs.fetch('brewfile_path', false)
+            "brew bundle#{inputs['verbose_log'] ? ' -vd' : nil} --file=#{inputs['brewfile_path']}"
+          else
+            # verbose_log defined, packages and use_brewfile/brewfile_path not defined - invalid
+            '# Invalid brew-install configuration!'
+          end
+        end
+
+        def translate_bundler(inputs)
+          return '# Invalid bundler step configuration!' unless required_bundler_inputs(inputs)
+
+          install_jobs = inputs['bundle_install_jobs']
+          install_retry = inputs['bundle_install_retry']
+
+          "bundle check || bundle install --jobs #{install_jobs} --retry #{install_retry}"
+        end
+
+        def required_bundler_inputs(inputs)
+          inputs.include?('bundle_install_jobs') && inputs.include?('bundle_install_retry')
+        end
+
         def translate_change_workdir(inputs)
           return '# Invalid change-workdir step configuration!' unless inputs.include?('path')
 
           [
-            inputs['is_create_path'] ? "mkdir #{inputs['path']}" : nil,
+            inputs.fetch('is_create_path', false) ? "mkdir #{inputs['path']}" : nil,
```

Comment:
this `fetch` is not necessary as the default value maps to the same conditional branch as when the key is not present (and resolves to `nil`)

Reply:
Switched